module/Application/src/Application/SchemaValidationException.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Application;
+
+class SchemaValidationException extends SchemaRetrievalException
+{
+    protected $validationMessages;
+
+    public function __construct($message = '', $code = 0, \Exception $previous = null, array $messages = []) {
```

Comment:
Want to swap the args around, like `($message = '', array $messages = [], $code = 0, \Exception $previous = null)`? For this exception we'll probably have `$messages` more often than `$code` or `$previous``